pombola/south_africa/views.py

```diff
@@ -1962,3 +1962,101 @@ def get_context_data(self, **kwargs):
             context['data'] = None
 
         return context
+
+class SAMpAttendanceView(TemplateView):
+    template_name = "south_africa/mp_attendance.html"
+
+    def calculate_abs_percenatge(self, num, total):
+        """
+        Return the truncated value of num, as a percentage of total.
+        """
+        return int("%.*f" % (0, (num / total * 100)))
```

Comment:
I don't think we've documented this anywhere, but we've generally been trying to use `.format` instead of `%` for string interpolation in Pombola